google/provider_test.go

```diff
@@ -40,41 +59,16 @@ func testAccPreCheck(t *testing.T) {
 		os.Setenv("GOOGLE_CREDENTIALS", string(creds))
 	}
 
-	multiEnvSearch := func(ks []string) string {
-		for _, k := range ks {
-			if v := os.Getenv(k); v != "" {
-				return v
-			}
-		}
-		return ""
-	}
-
-	creds := []string{
-		"GOOGLE_CREDENTIALS",
-		"GOOGLE_CLOUD_KEYFILE_JSON",
-		"GCLOUD_KEYFILE_JSON",
-		"GOOGLE_USE_DEFAULT_CREDENTIALS",
-	}
-	if v := multiEnvSearch(creds); v == "" {
-		t.Fatalf("One of %s must be set for acceptance tests", strings.Join(creds, ", "))
+	if v := multiEnvSearch(credsEnvVars); v == "" {
+		t.Fatalf("One of %s must be set for acceptance tests", strings.Join(credsEnvVars, ", "))
 	}
 
-	projs := []string{
-		"GOOGLE_PROJECT",
-		"GCLOUD_PROJECT",
-		"CLOUDSDK_CORE_PROJECT",
-	}
-	if v := multiEnvSearch(projs); v == "" {
-		t.Fatalf("One of %s must be set for acceptance tests", strings.Join(projs, ", "))
+	if v := multiEnvSearch(projectEnvVars); v == "" {
```

Comment:
Consider using the `getTestProjectFromEnv` you created below.

Reply:
I couldn't decide which was better; they are functionally equivalent but I felt the current implementation was less surprising since the others don't have get functions...

Which do you prefer?

Reply:
OK then I'll leave it as is, not because I think it's better but because I'm lazy